src/test/resources/setup.sh

```diff
@@ -71,10 +71,16 @@ oc new-app ${SYNDESIS_TEMPLATE_TYPE}  \
     -p OPENSHIFT_OAUTH_CLIENT_ID=$(oc project -q) \
     -n ${KUBERNETES_NAMESPACE}
 
-
-#Move image streams (one by one) inside the test namespace
-mkdir -p /tmp/syndesis-test-resources/
-for i in `oc get is -n syndesis-ci | grep -v NAME | cut -d" " -f1`; do
+# If env variable `SYNDESIS_DOCKERHUB_IMAGES` IS provided by template will be used
+if [ -n "${SYNDESIS_RELEASED_IMAGES}" ]; then
+  echo "ImageStreams specified by template will be used"
+  # no op required
+else
+  # Move image streams (one by one) inside the test namespace
+  echo "ImageStreams from CI namespace will be used"
+  mkdir -p /tmp/syndesis-test-resources/
+  for i in `oc get is -n syndesis-ci | grep -v NAME | cut -d" " -f1`; do
     oc export is $i -n syndesis-ci > /tmp/syndesis-test-resources/$i.yml
```

Comment:
I wonder if this whole block can all be simplified to `oc export is -n syndesis-ci | oc apply -f -` ? Could you try it out and see if it works?